frontend/src/components/phase-banner/index.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from '@emotion/styled'
+import { Text } from '../text'
+import tag from 'clean-tag'
+
+export const PhaseBannerContainer = styled(tag.div)`
```

Comment:
We'll be able to refactor this to a container component once I finish my next PR ;)